README.md

```diff
@@ -1,105 +1,256 @@
+    ```
+
+    The resoulte should looks like:
+
+    ```text
+    NAMESPACE    NAME                  AGE
+    kcp-system   moduletemplate-serverless   2m24s
+    ```
+
+10. Give Module Manager permission to install CustomResourceDefinition (CRD) cluster-wide
+
+    >**NOTE:** This is a temporary workaround and is only required in the single-cluster mode
+
+    Module-manager must be able to apply CRDs to install modules. In the remote mode (with control-plane managing remote clusters) it gets an administrative kubeconfig, targeting the remote cluster to do so. But in local mode (single-cluster mode), it uses Service Account and does not have permission to create CRDs by default.
+
+    Run the following to make sure the module manager's Service Account becomes an administrative role
+
+    ```bash
+    kubectl edit clusterrole module-manager-manager-role
+    ```
+
+    And add the following element under the `rules` field
+
+    ```yaml
+    - apiGroups:
+      - "*"
+      resources:
+      - "*"                  
+      verbs:                  
+      - "*"
+    ```
+
+11. Edit the Serverless in the Kyma CR
+
+    ```bash
+    kubectl edit kymas.operator.kyma-project.io -n kcp-system default-kyma
+    ```
+
+    And add the following field under the `spec` field
+
+    ```yaml
+      modules:
+      - name: serverless
+        channel: alpha
+    ```
+
+## Manual installation
+
+1. Clone project
+
+    ```bash
+    git clone https://github.com/kyma-project/serverless-manager.git && cd serverless-manager/
+    ```
+
+2. Set `serverless-manager` image name
+
+    ```bash
+    export IMG=<DOCKER_USERNAME>/custom-serverless-manager:0.0.1
+    ```
+
+3. Verify compability
+
+    ```bash
+    make test
+    ```
+
+4. Build and push image to registry
+
+    ```bash
+    make module-image
+    ```
+
+5. Deploy
+
+    ```bash
+    make deploy
+    ```
+
+## Using `serverless-manager`
+
+- Create Serverless instance
+
+    ```bash
+    kubectl apply -f config/samples/operator_v1alpha1_serverless_k3d.yaml
+    ```
+
+- Delete Serverless instance
+
+    ```bash
+    kubectl delete -f config/samples/operator_v1alpha1_serverless_k3d.yaml
+    ```
+
+- Update Serverless properties
+
+    This example shows how you can modify the Serverless docker registry address using the `serverless.operator.kyma-project.io` CR
+
+    ```bash
+    cat <<EOF | kubectl apply -f -
+    apiVersion: operator.kyma-project.io/v1alpha1
+    kind: Serverless
+    metadata:
+    name: serverless-sample
+    spec:
+    dockerRegistry:
+        enableInternal: false
+        registryAddress: k3d-kyma-registry:5000
+        serverAddress: k3d-kyma-registry:5000
+    EOF
+    ```
+
+## Troubleshooting
+
+- For MacBook M1 users
+
+    Some part of the scripts may not work because the `Kyma-Cli` is not released for Apple Silicon users. To fix it [install the Kyma-Cli manually](https://github.com/kyma-project/cli#installation) and export path to it
```

Comment:
```suggestion
    Some parts of the scripts may not work because the Kyma CLI is not released for Apple Silicon users. To fix it [install Kyma CLI manually](https://github.com/kyma-project/cli#installation) and export the path to it.
```